fe/src/components/Login.vue

```diff
@@ -34,12 +35,17 @@ export default {
   },
   methods: {
     submitForm() {
-      // not sure what to do here 
-      var res = httpService.post('api/user/create/', {email:'test@email.ca', password:'password'});
-      console.log(res)
-      console.log(res.data)
 
-    }  	
+      this.$refs['LoginForm'].validate((valid) => {
```

Comment:
this could be a constant

Reply:
this.$refs['Loginform'] ?? what's the benefit of that if you don't mind me asking